src/System/i18n/locales/en-US/translation.json

```diff
@@ -47,5 +47,13 @@
         "shipsFrom": "Ships from"
       }
     }
+  },
+  "artQuiz": {
```

Comment:
`artQuizPage`

Reply:
oh gotcha, also there are some conflicts with the translation json file in this pr that I'll handle and then its ready for merge

Reply:
i think ill also hook up any helper functions for translating if the functionality is there